core/vm/instructions.go

```diff
@@ -762,6 +852,9 @@ func opCall(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byt
 }
 
 func opCallCode(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
+	if !chargeCallVariantEIP4762(interpreter.evm, scope) {
```

Comment:
For the rest of `*CALL`, we simply call the helper so keeps being relatively simple.